### ISSUE TITLE
"Error: Malformed remote specification" with Bioconductor packages

### ISSUE THREAD

**Issue by KasperSkytte** [r-lib/remotes#134]
Hi!

As far as I can understand from [this](https://github.com/r-lib/devtools/blob/master/vignettes/dependencies.Rmd) vignette, if I want `remotes` to install a dependency (specifically `DESeq2`) from the `Bioconductor` repository, I need to add the package to the `Remotes` field in the `DESCRIPTION` file of [my package](https://github.com/MadsAlbertsen/ampvis2) like this: `bioc::3.7/DESeq2`. However, I'm getting the following error:

```r
remotes::install_github("madsalbertsen/ampvis2@dev")
Downloading GitHub repo madsalbertsen/ampvis2@dev
Error: Malformed remote specification 'bioc::3.7/DESeq2', error: object 'install_bioc' of mode 'function' was not found
error: Malformed remote specification 'bioc::3.7/DESeq2', error: object 'install_bioc' of mode 'function' was not found
```

I get this error no matter if installing on Windows 10, Ubuntu 18.04, or Mac OSX. If I instead use the usual bioconductor way:
```r
source("https://bioconductor.org/biocLite.R")
biocLite("madsalbertsen/ampvis2@dev")
```
...it works just fine, but I'd be glad to avoid `biocLite` if possible. I also tried adding a `biocViews` field, and then it finds and installs `DESeq2` just fine, but fails to find some of the dependencies of `DESeq2`, specifically `DelayedArray` and `GenomeInfoDbData`. 

Did I misunderstand how to enter `Bioconductor` dependencies in the `Remotes` field or is there a bug that needs to be fixed? 

Thanks in advance.

**Comment by jimhester** [comment]
remotes does not currently handle Bioconductor repositories like devtools does, you will need to use devtools to install this until remotes is updated.

**Comment by jimhester** [comment]
This is fixed by #135
